mayastor/src/rebuild.rs

```diff
@@ -385,3 +450,78 @@ impl RebuildOperations for RebuildJob {
         }
     }
 }
+
+impl RebuildJob {
+    fn start_all_tasks(&mut self) {
+        assert_eq!(self.tasks.active, 0, "{} active tasks", self.tasks.active);
+
+        for n in 0 .. self.tasks.total {
+            self.next = match self.send_segment_task(n) {
+                Some(next) => next,
+                None => break, /* we've already got enough tasks to rebuild
+                                * the bdev */
+            };
+
+            self.tasks.active += 1;
```

Comment:
We could move this into the above "Some" case. This would then be consistent with the layout of the "start_task_by_id" function below.

Reply:
Done